server/planning/feed_parsers/superdesk_planning_xml_test.py

```diff
@@ -73,10 +73,54 @@ def test_content(self):
         self.assertEqual(item[GUID_FIELD], "urn:newsml:stt.fi:20220506:581312")
         self.assertEqual(item[ITEM_TYPE], CONTENT_TYPE.PLANNING)
         self.assertEqual(item["state"], CONTENT_STATE.INGESTED)
+        self.assertEqual(item["firstcreated"], datetime(2022, 2, 16, 12, 18, 17, tzinfo=tzoffset(None, 7200)))
+        self.assertEqual(item["versioncreated"], datetime(2022, 2, 16, 12, 18, 17, tzinfo=tzoffset(None, 7200)))
+        self.assertEqual(item["planning_date"], datetime(2022, 5, 6, 0, 0, tzinfo=tzutc()))
         self.assertEqual(
             item["slugline"],
             "Miten valtiovarainministeriön ehdotuksen mukaan esimerkiksi puolustus saa lisärahoitusta?",
         )
+        self.assertEqual(
+            item["description_text"],
+            "Alustatalous on Tiekartaston valmistumisen jälkeen globaalisti jatkanut nopeaa ja voimakasta kasvuaan",
+        )
         self.assertEqual(item["ednote"], "Valtiovarainministeriön ehdotus toiseksi lisätalousarvioksi")
-        self.assertEqual(item["coverages"][0]["coverage_id"], "ID_TEXT_120190859")
-        self.assertEqual(item["coverages"][0]["planning"]["g2_content_type"], "text")
+
+        self.assertEqual(len(item["coverages"]), 2)
+
+        coverage = item["coverages"][0]
+        self.assertEqual(coverage["coverage_id"], "ID_TEXT_120190859")
+        self.assertEqual(coverage["workflow_status"], "draft")
+        self.assertEqual(coverage["firstcreated"], item["firstcreated"])
+        self.assertEqual(coverage["versioncreated"], datetime(2022, 5, 6, 2, 26, 27, tzinfo=tzoffset(None, 7200)))
```

Comment:
is this converted to utc at some point?

Reply:
Yes, looks like Superdesk/Eve data-layer is converting it to UTC:
![image](https://user-images.githubusercontent.com/22425232/181420827-95dd40a6-560a-49da-a0a1-20352c91ccdc.png)